### PR TITLE
chore: add nginx service prerender.io support

### PR DESCRIPTION
**Pull request by nermalcat69**
#### What Type of Change is this?

- [ ] New Page
- [ ] Minor Fix
- [x] Minor Improvement
- [ ] Major Improvement

#### Related issues & labels (optional)

- /closes #

### CODE REVIEW

File: apps/docs/content/nginx/faq.mdx [hunk 1]
```diff
@@ -7,4 +7,7 @@ import { FAQ, FAQItem } from '/src/components/Faq';
 
 <FAQ>
   <FAQItem question="Sample Question">sample answer</FAQItem>
```
Comment (vankovap) [inline apps/docs/content/nginx/faq.mdx]: Delete this please, when there is real content now

File: apps/docs/content/nginx/how-to/customize-web-server.mdx [hunk 2]
```diff
@@ -84,3 +84,9 @@ zerops:
 5. Ensure that the <a href="/nginx/how-to/build-pipeline#deployfiles">`build.deployFiles`</a> contains the folder with the `siteConfigPath` or add the path to the Nginx config file to the `deployFiles` list. Zerops will deploy the file to the runtime container(s).
 
 6. [Trigger](/nginx/how-to/trigger-pipeline) the build & deploy pipeline.
+
+### Built-in Prerender.io Support
+
+The default Nginx configuration includes automatic prerender.io support for SEO optimization. When `PRERENDER_TOKEN` is set, Nginx will automatically serve pre-rendered content to search engines and social media crawlers.
+
+See [environment variables](/nginx/how-to/env-variables#prenderio-support) for configuration details.
```
Comment (vankovap) [inline apps/docs/content/nginx/how-to/customize-web-server.mdx]: the link has a typo `#prenderio-support` -> `#prerenderio-support`

File: apps/docs/content/nginx/how-to/env-variables.mdx [hunk 3]
```diff
@@ -194,3 +194,45 @@ You can use runtime env variables in the build environment using the `RUNTIME_`
...
+      <th className="w-fit"><strong>Required</strong></th>
+      <th className="w-fit"><strong>Description</strong></th>
+      <th className="w-fit"><strong>Default</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr className="text-center">
+      <td className="w-fit">PRERENDER_TOKEN</td>
+      <td className="w-fit">Yes</td>
+      <td className="w-fit">Your prerender.io service token (set as secret)</td>
```
Comment (vankovap) [inline apps/docs/content/nginx/how-to/env-variables.mdx]: you can skip the `(set as secret)`, it's in the tip below :+1:

**Review by vankovap — COMMENTED** [review]
:tada: just minor things

File: apps/docs/content/nginx/how-to/env-variables.mdx (hunk 3 above)
Comment (nermalcat69) [reply apps/docs/content/nginx/how-to/env-variables.mdx]: Done!

File: apps/docs/content/nginx/how-to/customize-web-server.mdx (hunk 2 above)
Comment (nermalcat69) [reply apps/docs/content/nginx/how-to/customize-web-server.mdx]: Done!

File: apps/docs/content/nginx/faq.mdx (hunk 1 above)
Comment (nermalcat69) [reply apps/docs/content/nginx/faq.mdx]: Done!

**Review by vankovap — APPROVED** [review]

**Pull request merged**